### PR TITLE
Add Ghommal's Hilt Teleport Locations on World Map

### PR DESCRIPTION
**Pull request by KPGoik**
Resolves Issue [#16006](https://github.com/runelite/runelite/issues/16006).

### CODE REVIEW

**Comment by KPGoik** [comment]
Didn't get ahead of myself this time waiting for the tests to go through...

Done! This resolves issue #16006

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java [hunk 1]
```diff
@@ -146,6 +146,8 @@
 	TELEPORT_CRYSTAL_PRIFDDINAS(TeleportType.OTHER, "Teleport crystal", "Prifddinas", new WorldPoint(3264, 6065, 0), "teleport_crystal_icon.png"),
 	DRAKANS_MEDALLION_VER_SINHAZA(TeleportType.OTHER, "Drakan's medallion", "Ver Sinhaza", new WorldPoint(3649, 3230, 0), "drakans_medallion_teleport_icon.png"),
 	DRAKANS_MEDALLION_DARKMEYER(TeleportType.OTHER, "Drakan's medallion", "Darkmeyer", new WorldPoint(3592, 3337, 0), "drakans_medallion_teleport_icon.png"),
+	GHOMMALS_HILT_GWD(TeleportType.OTHER, "Ghommal's hilt 1", "God Wars Dungeon", new WorldPoint(2898, 3709, 0), "ghommals_hilt_1_teleport_icon.png"),
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java:149]:
```suggestion
	GHOMMALS_HILT_GWD(TeleportType.OTHER, "Ghommal's hilt", "God Wars Dungeon", new WorldPoint(2898, 3709, 0), "ghommals_hilt_teleport_icon.png"),
```

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java [hunk 2]
```diff
@@ -146,6 +146,8 @@
...
+	GHOMMALS_HILT_MORULREK(TeleportType.OTHER, "Ghommal's hilt 4", "Mor Ul Rek", new WorldPoint(2554, 5130, 0), "ghommals_hilt_4_teleport_icon.png"),
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/TeleportLocationData.java:150]:
```suggestion
	GHOMMALS_HILT_MORULREK(TeleportType.OTHER, "Ghommal's hilt", "Mor Ul Rek", new WorldPoint(2554, 5130, 0), "ghommals_hilt_teleport_icon.png"),
```

**Review by raiyni — CHANGES_REQUESTED** [review]
We don't differentiate between different diary reward levels for other items.

**Comment by ragnarokatz** [comment]
do you mind if I take over and continue to work on this issue?

**Comment by raiyni** [comment]
@ragnarokatz go for it

**Comment by raiyni** [comment]
added in 9f3b6958fba80923c2926acd5ef3735b77b3e15b

**Pull request closed without merging**